packages/migrations/src/Command/MigrateCommand.php

```diff
@@ -107,10 +114,14 @@ function ($message) use ($output) {
             }
         );
 
-        $migrationsConfiguration = new Configuration($this->em->getConnection(), $outputWriter);
+        $migrationsConfiguration = new Configuration($this->migrationsConfig, $this->em->getConnection(), $outputWriter);
         $configurationHelper = new ConfigurationHelper($this->em->getConnection(), $migrationsConfiguration);
         $this->getApplication()->getHelperSet()->set($configurationHelper, 'configuration');
 
+        $migrationsConfiguration->setMigrationsDirectory(self::MIGRATIONS_DIRECTORY);
+        $migrationsConfiguration->setMigrationsNamespace(self::MIGRATIONS_NAMESPACE);
+        $migrationsConfiguration->setMigrationsFinder($this->migrationFinder);
```

Comment:
If you don't need to change this in runtime, why don't you set this in the DIC configuration?

Reply:
We create the instance of Configuration on our own, so it cannot be injected, but I have moved this to parameters passed to the constructor instead of setting it via the method.